code/modules/mob/living/simple_animal/hostile/illusion.dm

```diff
@@ -58,7 +58,7 @@
 			return
 		var/mob/living/simple_animal/hostile/illusion/M = new(loc)
 		M.faction = faction.Copy()
-		M.Copy_Parent(parent_mob, life_span, health/2, melee_damage_upper, multiply_chance)
+		M.Copy_Parent(parent_mob, 80, health/2, melee_damage_upper, multiply_chance)
```

Comment:
but I want them to replicate forever if you don't kill them/move away for ~7 seconds

It won't run into the previous problem of infinite clones though since they will actually despawn properly now